mysite/code/BlocksPage.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+class BlocksPage extends Page
```

Comment:
If this is to be a thing by default, should `dnadesign/silverstripe-elemental-userforms` not also be shifted from a `suggest` to a `require` in the composer.json?

Reply:
I'm not sure I follow your reasoning

Reply:
Sorry, I was confused between content blocks and userforms. Ignore! :>